src/riak_kv_qry_worker.erl

```diff
@@ -261,6 +261,11 @@ throttling_spawn_index_fsms(State) ->
 estimate_query_size(#state{n_subqueries_done = NSubqueriesDone} = State)
   when NSubqueriesDone < 2  ->
     State;
+estimate_query_size(#state{qry = ?SQL_SELECT{'SELECT' = #riak_sel_clause_v1{calc_type = Type}}} = State)
+  when Type /= rows ->
```

Comment:
Any reason to not estimate grouped rows? The result can be as big as the full rows.